torch/csrc/jit/codegen/cuda/runtime/bf16_support.cu

```diff
@@ -17,18 +17,98 @@ struct __align__(2) __bfloat {
   unsigned short __x;
 };
 
-__device__ __bfloat __float2bfloat(const float f) {
+__device__ __inline__ __bfloat __float2bfloat(const float f) {
   __bfloat val;
   asm("{  cvt.rn.bf16.f32 %0, %1;}\n"
       : "=h"(__NVFUSER_BFLOAT_TO_US(val))
       : "f"(f));
   return val;
 }
 
-__device__ float __bfloat2float(const __bfloat h) {
+__device__ __inline__ __bfloat __double2bfloat(const double d) {
+#if __CUDA_ARCH__ >= 700
+  __bfloat val;
+  asm("{  cvt.rn.bf16.f64 %0, %1;}\n"
```

Comment:
```
.relu modifier and {.f16x2, .bf16, .bf16x2, .tf32} destination formats require sm_80 or higher.

cvt.bf16.{u8/s8/u16/s16/u32/s32/u64/s64/f16/f64/bf16}, cvt.{u8/s8/u16/s16/u32/s32/u64/s64/f16/f64}.bf16, and cvt.tf32.f32.{relu}.{rn/rz} require sm_90 or higher.
```
I see we have sm_70 in the requirement. :fearful: This doesn't seem right.

Reply:
Reference: https://docs.nvidia.com/cuda/parallel-thread-execution/index.html#data-movement-and-conversion-instructions-cvt
`Target ISA Notes`

Reply:
Thanks, copy and paste got the better of me.

Reply:
I fixed this